src/OutputFormat/CalculationBehaviour.jl

```diff
@@ -5,6 +5,10 @@ using .InputParser
 
 export CalculationOutputBehaviour, ConsolidationSwellCalculationBehaviour, LeonardFrostCalculationBehaviour, SchmertmannCalculationBehaviour, SchmertmannElasticCalculationBehaviour, CollapsibleSoilCalculationBehaviour, writeCalculationOutput, getCalculationOutput, getCalculationValue
 
+# For now this will be hardcoded into here, later it will
+# rely on our choice of units
+GAMMA_W = 0.03125
```

Comment:
I think we should have a separate module for constants.  We also want to think about how we handle units.  We could have a configuration parameter that determines the units for each problem.  Since the imperial units require the divide by g at points, the equations do change a little bit.  We probably can have a standard set of units for imperial and metric.